packages/documentation-site/patternfly-docs/content/get-started/release-highlights.md

```diff
@@ -3,10 +3,163 @@ id: Release highlights
 title: Release highlights
 section: get-started
 ---
-
 import './get-started.css';
 import { Divider, Timestamp } from '@patternfly/react-core';
 
+<Timestamp>March 2024</Timestamp>
```

Comment:
Since we're using the Timestamp component, we should pass in the `date` property. Without it, the `datetime` attribute on the `<time>` element will be the current datetime based on when the page was rendered, so come April the datetime would refer to April rather than March. This should do it:

```suggestion
<Timestamp date={new Date(2025, 2, 19)}>March 2024</Timestamp>
```

The 19 can be changed to whatever date the PR would actually be merged.